2022-12-16 17:52:42,038-op3.py[line:76]-WARNING:连接对象创建成功
2022-12-16 17:52:46,922-op3.py[line:93]-WARNING:添加点名成功
2022-12-16 17:57:26,956-op3.py[line:153]-WARNING:read history data failed
2022-12-16 17:57:26,971-op3.py[line:204]-WARNING:关闭连接成功
2022-12-16 17:59:38,065-op3.py[line:76]-WARNING:连接对象创建成功
2022-12-16 17:59:39,481-op3.py[line:93]-WARNING:添加点名成功
